config/test-bluetooth_signal.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "wifi": {
+          "driver": "subt.bluetooth_signal:Bluetooth_signal",
```

Comment:
the class name should be CamelCase, i.e. BluetoothSignal ... and I would maybe just use shorter version `bluetooth:Bluetooth`